### ISSUE TITLE
Agent does not show up in Plex settings

### ISSUE THREAD

**Issue by Robinfr** [ZeroQI/YouTube-Agent.bundle#71]
I've downloaded the ZIP file, extracted it, renamed it to `YouTube-Agent.bundle` and copied that to my Plex `Plug-Ins` folder on my server, yet even after making sure the permissions and owner are correctly set, and restarting multiple times, the agent does not show up in the settings. I've done this process several times, but to no avail. 

Is there something else that I should be doing to get the agent to show up in the list of agents?

**Comment by Robinfr** [comment]
Yes I tried https://github.com/JordyAlkema/Youtube-DL-Agent.bundle which does in fact show up. I'll see if I can try again today and see if there is anything in the logs.

**Comment by ZeroQI** [comment]
com.plexapp.system.log should have the answer, and possibly missing the plex token and the fallback doesn't work in your case

**Comment by Robinfr** [comment]
There was indeed an error in there:

```
2021-02-24 11:18:50,833 (14dc47bcaf40) :  CRITICAL (core:569) - Exception while loading code (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/core.py", line 563, in load_code
    self.init_code = self.loader.load(self.init_path, elevated, use_xpython = Framework.constants.flags.use_xpython in self.sandbox.flags)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/loader.py", line 47, in load
    code = self.compile(str(source), str(uni(filename)), elevated)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/loader.py", line 52, in compile
    return RestrictedPython.compile_restricted(source, name, 'exec', elevated=elevated)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/RestrictedPython/RCompile.py", line 115, in compile_restricted
    gen.compile()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/RestrictedPython/RCompile.py", line 68, in compile
    tree = self._get_tree()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/RestrictedPython/RCompile.py", line 59, in _get_tree
    tree = self.parse()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/RestrictedPython/RCompile.py", line 56, in parse
    return niceParse(self.source, self.filename, self.mode)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c1c83ba4/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/RestrictedPython/RCompile.py", line 38, in niceParse
    compile(source, filename, mode)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/YouTube-Agent.bundle/Contents/Code/__init__.py", line 310
    i thumb and thumb in episode.thumbs:
          ^
SyntaxError: invalid syntax

2021-02-24 11:18:50,834 (14dc40702700) :  DEBUG (core:538) - Machine identifier is 429e6cdbeda054aa79c7fa0b41446c4905f61316
2021-02-24 11:18:50,835 (14dc40702700) :  DEBUG (core:539) - Server version is 1.21.3.4046-3c1c83ba4
```

**Comment by Robinfr** [comment]
Created a PR to fix it #72

**Comment by ZeroQI** [comment]
My bad. Thanks for the fix
